.circleci/config.yml

```diff
@@ -556,6 +557,19 @@ jobs:
           command: |
             python3 tools/deploy_s3.py dist-pyc/ "dev/pyc/" --bucket "pyodide-cdn2.iodide.io" --cache-control 'max-age=3600, public' --rm-remote-prefix
 
+      - run:
+          # Unlike the release version, we upload the dev version to S3 not to GitHub.
+          name: Deploy release files to S3
+          command: |
+            mkdir -p /tmp/ghr/dist
+            cp -r dist /tmp/ghr/pyodide
+            cp -r xbuildenv /tmp/ghr/xbuildenv
+            cd /tmp/ghr
+            tar cjf dist/pyodide-core.tar.bz2 pyodide/pyodide{.js,.mjs,.asm.js,.asm.wasm} pyodide/{package,repodata}.json pyodide/python_stdlib.zip
+            tar cjf dist/xbuildenv.tar.bz2  xbuildenv/
```

Comment:
Ahh, I think we haven't whitelisted this extension with jsdelivr so it won't work. 

Maybe we could put it to `pyodide-cache` bucket where it was originally,
```
python3 tools/deploy_s3.py dist/ "xbuildenv/" --bucket "pyodide-cache" --cache-control 'max-age=3600, public' --rm-remote-prefix
```
but then we need to pass it different credentials somehow. For instance maybe define `AWS_ACCESS_KEY_ID_CACHE`,  `AWS_SECRET_ACCESS_KEY_CACHE` (sent you by email) and then set them back to the variables without the `_CACHE` prefix just for the below command.

Reply:
Thanks, I changed the bucket and added the corresponding keys.